tests/content_visibility/delete_post_test.php

```diff
@@ -308,11 +311,14 @@ public function test_delete_post($forum_id, $topic_id, $post_id, $data, $is_soft
 			)));
 		$user = new \phpbb\user('\phpbb\datetime');
 
-		$phpbb_dispatcher = new phpbb_mock_event_dispatcher();
```

Comment:
As per the other PR, this should be used, instead of `$phpbb_dispatcher = $this->getMockBuilder('\phpbb\event\dispatcher')...`
Just move this up to the first mock creation